### ISSUE TITLE
Map Not Loading

### ISSUE THREAD

**Issue by JayJalie** [JayJalie/OperonDigital#1]
I am a newbie at this. Started learning about using Leaflet for a project; however, my issue is that the map will not load. The idea is to get the map loaded properly on the Gitpage, only then I can develop further. Thanks.

<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
    <link rel="stylesheet" href="/css/leaflet.css">

    <title>Operon Insights</title>

    <script src="/js/leaflet.js"></script>
    <script src="/js/jquery-3.6.0.min.js"></script>
    <script src="/js/leaflet.ajax.min.js"></script>
    

    <style>
        body {
            padding: 0;
            margin: 0;
        }
        html, body, #map {
            height: 100%;
            width: 100%;
        }
   
    </style>

</head>
<body>
    
    <div id="map"></div>

    <Script>
    const here = {
        apiKey:'5_5YsZj0iAT9l7x1doLzXB85NvVUk4ZHOFvs4pORm9s'
        }
        const style = 'reduced.day';
  
    const hereTileUrl = `https://2.base.maps.ls.hereapi.com/maptile/2.1/maptile/newest/${style}/{z}/{x}/{y}/512/png8?apiKey=${here.apiKey}&ppi=320`;
  
    const map = L.map('map', {
         center: [3.162170, 101.604208],
         zoom: 15,
         layers: [L.tileLayer(hereTileUrl)]
    });
    map.attributionControl.addAttribution('&copy; HERE 2019, Operon Digital');
    </Script>

    <script src="main.js"></script>
    <script src="legend.js"></script>
 
    <style>
          .legend {
            width: 150px;
            font-size: 14px;
            color: #ffffff;
            font-family: "Open Sans", Helvetica, sans-serif;
            padding: 8px 10px;
            background-color: rgba(15, 153, 218, 0.8);
            box-shadow: 0 0 15px rgba(255, 255, 255, 0.2);
            border-radius: 5px;
            border: 0.1px solid grey;
        }    
    </style>

    
</body>
</html>
